app.html

```
@@ -18,8 +18,8 @@
       var canvas = document.getElementById('board');
       var board = new Board();
 
-      canvas.width = 600;
-      canvas.height = 600;
+      canvas.width = board.BOARD_SIZE;
+      canvas.height = board.BOARD_SIZE;
```

Yeah, one of the many, MANY annoying things about JS in the client: that value would simply be NULL, which just doesn't bother raising an exception. So it's a silent fail.